more/free-programming-cheatsheets.md

```diff
@@ -378,6 +379,11 @@
 * [TensorFlow v2.0 Cheat Sheet](https://web.archive.org/web/20200922212358/https://www.aicheatsheets.com/static/pdfs/tensorflow_v_2.0.pdf) - Altoros (PDF) *(:card_file_box: archived)*
 
 
+### TypeScript
+
+* [TypeScript Cheatsheet](https://f001.backblazeb2.com/file/frontend-in-four-hours/BONUS+Typescript+cheat+sheet.pdf) - Jon Hilton (PDF)
```

Comment:
how do we know this is an authorized distribution?

Reply:
I wasn't able to find evidence that this is an authorized distribution by the author. I can either edit the pull request or create a new one replacing the pdf with an html TypeScript cheatsheet.